src/main/resources/db_script/user_card_creation.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE user_card (
```

Comment:
Why do we use NULL as a default value in DB. In this case we should write extra lines of code to check the NPE.
Could we find another approach ?